tools/python/util/check_onnx_model_mobile_usability.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import argparse
+import logging
+import pathlib
+
+# need this before the mobile helper imports for some reason
+logging.basicConfig(format='%(levelname)s:  %(message)s')
+
+from .mobile_helpers import check_model_can_use_ort_mobile_pkg, usability_checker  # noqa
+
+
+def check_usability():
+    parser = argparse.ArgumentParser(
+        description='''Analyze an ONNX model to determine how well it will work in mobile scenarios, and whether
+        it is likely to be able to use the pre-build ONNX Runtime Mobile Android or iOS package.''',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+
+    parser.add_argument('--config_path',
+                        help='Path to required operators and types configuration used to build '
+                             'the pre-built ORT mobile package.',
+                        required=False,
+                        type=pathlib.Path,
+                        default=check_model_can_use_ort_mobile_pkg.get_default_config_path())
+    parser.add_argument('--log_level', choices=['debug', 'info', 'warning', 'error'],
+                        default='info', help='Logging level')
+    parser.add_argument('model_path', help='Path to ONNX model to check', type=pathlib.Path)
+
+    args = parser.parse_args()
+    logger = logging.getLogger('default')
```

Comment:
>default [](http://example.com/codeflow?start=32&length=7)

what's the reason for using the logger name "default"?

Reply:
any name would do. cut-and-pasted from example code.

Reply:
would something like "check_usability" be more informative here?